pkgs/applications/networking/browsers/firefox/default.nix

```diff
@@ -64,7 +65,6 @@ stdenv.mkDerivation rec {
       "--disable-necko-wifi" # maybe we want to enable this at some point
       "--disable-installer"
       "--disable-updater"
-      "--disable-pulseaudio"
```

Comment:
Yes, but firefox doesn't link to pulseaudio and instead searches for the dynamic library. So if you don't have pulseaudio in `LD_LIBRARY_PATH` set by Firefox wrapper (with my patch this is so if you don't have `config.pulseaudio = true`), it won't use it.